dracula-theme.el

```diff
@@ -56,7 +55,8 @@
    'dracula
    ;; default
    `(cursor ((,class (:background ,fg3))))
-   `(default ((,class (:background ,bg1 :foreground ,fg1))))
+   `(default ((((type nil)) (:background "#000000" :foreground ,fg1))
```

Comment:
Sorry, had to edit. The correct term would be *short-cut* semantics, not *fall-through*, which was the term I used accidentally, and which is exactly the opposite.